Microsoft.RestApi.RestTransformer/Utility.cs

```diff
@@ -33,6 +33,26 @@ public static T GetValueFromMetaData<T>(this Dictionary<string, object> metadata
             return default(T);
         }
 
+        public static T GetValueFromMetaDataWithDefaultValue<T>(this Dictionary<string, object> metadata, string key, T defaultValue)
```

Comment:
remove the method, and add (T defaultValue = null) to the original GetValueFromMetaData method, to reduce the code duplication.

Reply:
Merged.

Reply:
Merged